lib/services/v2.0/nomination_pools/interfaces.dart

```diff
@@ -216,9 +213,6 @@ mixin KC2NominationPoolsInterface on ChainApiProvider {
     }
   }
 
-  /// @HolyGrease - is there an api got info of all bonded validators that can be nominated?
-  /// if no - we need to have it so user sees options to nominate...
-
```

Comment:
Yes, we have. Please search for `getValidators` method that returns `ValidatorPrefs`